lib/get-current-repo.js

```diff
@@ -0,0 +1,20 @@
+const path = require('path')
+const fs = require('fs')
+
+module.exports = (currentProcess = process) => {
+  // Default output
+  let name = ''
```

Comment:
I would suggest defaulting to `null`, that way you can later check for the presence of the repo more simply: `if (currentRepo)`